sentinel-demo/sentinel-demo-slot-chain-spi/src/main/java/com/alibaba/csp/sentinel/demo/slot/DemoSlotChainBuilder.java

```diff
@@ -20,17 +20,11 @@
 import com.alibaba.csp.sentinel.slots.DefaultSlotChainBuilder;
 
 /**
- * An example slot chain builder. To activate this slot chain builder,
- * add the class name to corresponding SPI file in `resource/META-INF/services` directory.
- *
  * @author Eric Zhao
+ *
+ * @deprecated since 1.7.2, we can use @SpiOrder(-3500) to adjust the order of {@link DemoSlot},
+ * this class is reserved for compatibility with older versions.
```

Comment:
If has no annotation, the order is `SpiOrder.LOWEST_PRECEDENCE` and the slot will be at the end of slot chain.
How about adding `@SpiOrder(-3500)` in `DemoSlot` class?  
As the comment mentioned, the `DemoSlot` will be after `SystemSlot` and before `FlowSlot`,  and the demo is more friendly for new users.

Reply:
good idea,done